README.md

```diff
@@ -5,6 +5,30 @@ CV management system
 
 Master branch auto-deploys to <a href="http://cv-master.herokuapp.com">cv-master.herokuapp.com</a>
 
+##  Heroku setup
+Heroku should be provisioned with a postgres-database, and sets up the tables automatically when deployed to heroku.
+
+```
+heroku create
+heroku addons:create heroku-postgresql:hobby-dev -a {app name}
+heroku git:remote -a {app name}
+cat SQL_table_creation.sql | heroku pg:psql -a {app name}
+git push heroku
+heroku open
+```
+
+By default the template-styling for pdf-generation is loaded to the database on dyno startup, according to the files in /server/pdf/. You can opt to disable this and update templates to the database manually by setting the env flag UPDATE_PDF_FROM_FS to 0.
+
+### Config variables for heroku deployment
+```
+  // Needed for authentication
+  AUTH_ID:  
```

Comment:
Mitä auth_id ja auth_secrettiin kuuluu kirjottaa vai kirjotetaanko mitään ?

Reply:
Ne saa OAuthista, et sen pitäis olla aika self-explanatory jos on sellane. Mut voisin laittaa siihen jonku kommentin kuitenki.

Reply:
Vois ehkä olla ihan hyvä kirjottaa et ihan tietämättömätki ymmärtää  : D